pkg/config/config.go

```diff
@@ -1540,6 +1542,11 @@ func setupFipsEndpoints(config Config) error {
 
 	// HTTP for now, will soon be updated to HTTPS
 	protocol := "http://"
+	if config.GetBool("fips.https") {
+		protocol = "https://"
+		// top-level setting override precedence
```

Comment:
Might be me but I feel like this comment is a bit vague. I first understood that the top-level setting has priority. Would `tls_verify overrides top-level setting precedence` be better ?